Yoast/wordpress-seo
# FRO-16 update styling table

## Context


* Update the styling of the bulk editor tables to match the new Yoast styling

## Summary


This PR can be summarized in the following changelog entry:

* Update the styling of the bulk editor tables to match the new Yoast styling

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch
* Checkout 14297-update-styling-table from the javascript repo
* Link the javascript repo
* Build the plugin
* Check if the bulk editor tables in the tools section match the new design

## UI changes

## Documentation

## Quality assurance


Fixes https://github.com/Yoast/wordpress-seo/issues/14297
